docs/navigation-structure.md

```diff
@@ -38,6 +38,12 @@ nav_order: 4
 ---
 ```
 
+The specified `nav_order` parameters on a site should be all integers or all strings.
+Pages without a `nav_order` parameter are ordered alphabetically by their `title`,
+and appear after the explicitly-ordered pages at each level.
+To sort all Capital letters before lowercase letters,
```

Comment:
In connection with the proposed change to the default for `nav_sort`, the documentation here needs updating, e.g.:

```suggestion
By default, all Capital letters are sorted before all lowercase letters;
adding `nav_sort: case_insensitive` in the configuration file ignores case
when sorting strings (but also sorts numbers lexicographically: `10` comes before `1`).
```